generator/src/test/java/com/scottlogic/deg/generator/cucumber/utils/CucumberGenerationConfigSource.java

```diff
@@ -85,6 +84,11 @@ public boolean shouldViolate() {
         return state.shouldViolate;
     }
 
+    @Override
+    public boolean overwriteOutputFiles() {
+        return state.overwriteOutputFiles;
```

Comment:
We dont need to be able to vary this in cucumber, we could always return `false`

Reply:
changed to return false